packages/components/src/Tree/Tree.hooks.ts

```diff
@@ -91,10 +91,12 @@ function makeDetailedItems(
 ): TreeItem[] {
 	return items.map((item) => {
 		const { children, onClick, onExpand, selected, expanded, ...rest } = item
+		const itemSelected =
```

Comment:
let's streamline this a bit, and rename `itemSelected` to `selected`, and include item.selected, eeg.

```js
const { children, onClick, onExpand, expanded, selected: itemSelected...rest } = item
const selected = itemSelected || item.key === selectedKey || selectedKey?.includes(`${item.key}`/)

...
```

Reply:
Yeah I think startsWith makes sense to use as well, I could see cases where the key is in the middle of another matched route for some reason, causing some jank selection behavior.